code/messaging/build.gradle

```diff
@@ -182,11 +207,12 @@ dependencies {
     // where Objenesis 2 requires minimum Android API 26
     androidTestImplementation 'org.powermock:powermock-module-junit4:1+'
     androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.9.9'
-    androidTestImplementation 'com.adobe.marketing.mobile:edgeidentity:1.0.0'
+    androidTestImplementation('com.adobe.marketing.mobile:edgeidentity:1.0.0')
```

Comment:
nit: Let's keep it consistent with other dependencies and remove the brackets `()`.